### PR TITLE
translation to Dutch

### PR DESCRIPTION
**Pull request by Haroenv**
see #54

### CODE REVIEW

**Comment by Maddoc42** [comment]
Awesome, keep them coming! :+1:

**Comment by Haroenv** [comment]
Done now, should probably be squashed, but you can do that while merging. 

There's one typo in the names, but I just put a comment, I didn't fix it.

File: client/app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,295 @@
+<resources>
+    <string name="app_name">MrHyde</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-nl/strings.xml]: Please remove this line, we won't translate MrHyde in the future :)

File: client/app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    <string name="action_publish_draft">Publiceer draft</string>
+    <string name="action_unpublish_post">Verberg post</string>
+    <string name="action_insert_markdown">Voeg markdown in</string>
+    <string name="action_open_in_browser">Open in browser</string>
+    <string name="action_help">Help</string>
+
+    <string name="select_git_hoster_title">Kies een provider</string>
+    <string name="github">GitHub</string>
+    <string name="bitbucket">Bitbucket</string>
+    <string name="gitlab">GitLab</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-nl/strings.xml]: Please remove this and the two above lines. No translation needed.

File: client/app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+    <string name="drafts">Drafts</string>
+    <string name="draft">Draft</string>
+    <string name="new_draft">Nieuwe draft</string>
+    <string name="no_drafts">Nog geen drafts</string>
+    <string name="all_files">Alle bestanden</string>
+    <string name="view_all_files">Bekijk alle bestanden</string>
+
+    <string name="filename">Bestandsnaam</string>
+    <string name="your_awesome_title">Je supertitel</string>
+    <string name="default_front_matter">---\nlayout: post\ntitle: %1$s\n---\n</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-nl/strings.xml]: Please remove. Default front matter will stay the same for all languages.

File: client/app/src/main/res/values-nl/strings.xml [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+        &lt;br/&gt;
+        Philipp Eichhorn&lt;br/&gt;
+        Simon Hofmann&lt;br/&gt;
+        Julia Schottenhamml&lt;br/&gt;
+        Richard Baumann&lt;br/&gt;
+        &lt;br/&gt;
+        Veel codeplezier!
+    </string>
+    <string name="credits">Credits</string>
+    <string name="credits_msg">
```
Comment (Maddoc42) [inline client/app/src/main/res/values-nl/strings.xml]: Please remove. If not translated we don't want to track changes in multiples places.

File: client/app/src/main/res/values-nl/strings.xml [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+    <string name="markdown_h3">H3 hoofding</string>
+    <string name="markdown_h4">H4 hoofding</string>
+    <string name="markdown_h5">H5 hoofding</string>
+    <string name="markdown_h6">H6 hoofding</string>
+    <string name="markdown_ul">Lijst van selection</string>
+    <string name="markdown_ol">Georderde lijst van selection</string>
+    <string name="markdown_quote">Quote van selection</string>
+    <string name="markdown_code">Codeblok van selection</string>
+
+    <string name="markdown_element_bold">**</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-nl/strings.xml]: Please remove the `markdown_element_*` entries, as they are markdown elements that are not translated.

**Comment by Haroenv** [comment]
Sure, I'll remove the Ones that aren't translated, but I remember from when I did that in another app you needed to translate everything. Is this changed recently?

**Comment by Haroenv** [comment]
I'm still kinda doubting to translate `draft`. Now I kept it English, because I can't find a suitable translation, but ideally it should be translated. I'll ask around maybe

**Comment by Haroenv** [comment]
Translated `drafts` as `concept`, because that's how Mail.app translates it.

**Comment by Maddoc42** [comment]
Android uses the values under `res/values/strings.xml` as a default. Whenever a localized version of a key is present that one will be used instead. :space_invader: 

Cheers, changes look good! :muscle: thanks again.

Do you mind being mentioned in the credits / about section?

**Comment by Haroenv** [comment]
That would be nice. Thanks for the app!

**Pull request merged**